.env

```
@@ -0,0 +1 @@
+VITE_API_BASE_URL=http://localhost:8080
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)